paper/paper.md

```diff
@@ -80,11 +80,11 @@ Firstly, it includes VILLASweb, a tool designed for the visualization and monito
 Secondly, there is VILLASfpga, which provides hard real-time capabilities for VILLASnode.
```

Comment:
```suggestion
VILLASnode also supports interfacing with FPGAs, thus enabling hard real-time communication of software with arbitrary device connected to the FPGA.
```
VILLASfpga has been fully integrated into VILLASnode. (sorry for suggesting something that has not been touched)